### ISSUE TITLE
Remote server list appear when selecting local

### ISSUE THREAD

**Issue by noplay** [GNS3/gns3-gui#604]
https://github.com/GNS3/gns3-gui/commit/bb131b4ff51a4a2b6e3bcf363eb51e6700025b46#commitcomment-12582524

**Comment by noplay** [comment]
I will move VirtualBox and VMware to the vm_wizard in order to have all the wizards using the same code.

**Comment by noplay** [comment]
I hope I didn't break something. But it's cleaner now all the common code is in the same place. We should no longer have different behavior between the wizards even for VirtualBox and VMware

**Comment by ehlers** [comment]
Well, basically it works, but I have found two new issues:

- The VMware wizard shows an empty list of remote servers, when selecting server type "Remote". Before your change it shows the list of the remote servers. The VirtualBox wizard does it right (before and after the change).
- The VirtualBox wizard creates exceptions about a QRadioButton object:
```
Exception detected, traceback information saved in /Users/behlers/.config/GNS3/exceptions.log ******

PLEASE REPORT ON https://community.gns3.com/community/software/bug

Traceback (most recent call last):
  File "/usr/local/lib/python3.4/site-packages/gns3_gui-1.4.0.dev5-py3.4.egg/gns3/modules/virtualbox/dialogs/virtualbox_vm_wizard.py", line 67, in initializePage
    super().initializePage(page_id)
  File "/usr/local/lib/python3.4/site-packages/gns3_gui-1.4.0.dev5-py3.4.egg/gns3/dialogs/vm_wizard.py", line 117, in initializePage
    elif self._use_local_server and self.uiLocalRadioButton.enabled():
AttributeError: 'QRadioButton' object has no attribute 'enabled'
```

**Comment by noplay** [comment]
Should be fixed by 8b38d4967cf8da362961844128d8c8b599282769 but require testing before releasing it to everyone

**Comment by ehlers** [comment]
I haven't found any errors in the GUI.

As I'm not using VMware, I can't really test the wizard. But I have tested the VirtualBox wizard with local and remote server and it looks good. The same is true for the Qemu wizard.
